src/lib.rs

```diff
@@ -371,6 +355,21 @@ impl Pixels {
             pos.1.max(0).min(self.texture_extent.height as isize - 1) as usize,
         )
     }
+
+    /// Provides access the the [`wgpu::Device`] pixels uses.
```

Comment:
In this three doc comments, we should either put `pixels` in backticks to refer to the crate, or reword the documentation so it doesn't need to refer to itself by name.

Reply:
Sure.